streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -639,16 +643,16 @@
                     Serdes.ByteArraySerde.class.getName(),
                     Importance.MEDIUM,
                     DEFAULT_VALUE_SERDE_CLASS_DOC)
-            .define(NUM_STANDBY_REPLICAS_CONFIG,
-                    Type.INT,
-                    0,
+            .define(DEFAULT_WINDOWED_KEY_SERDE_INNER_CLASS,
+                    Type.CLASS,
+                    null,
                     Importance.MEDIUM,
-                    NUM_STANDBY_REPLICAS_DOC)
-            .define(NUM_STREAM_THREADS_CONFIG,
-                    Type.INT,
-                    1,
+                    DEFAULT_WINDOWED_KEY_SERDE_INNER_CLASS_DOC)
+            .define(DEFAULT_WINDOWED_VALUE_SERDE_INNER_CLASS,
+                    Type.CLASS,
+                    null,
                     Importance.MEDIUM,
-                    NUM_STREAM_THREADS_DOC)
+                    DEFAULT_WINDOWED_VALUE_SERDE_INNER_CLASS_DOC)
```

Comment:
This is all newly added: we weren't defining these configs before at all, which means they had no default set, no description, were left out of the autogenerated docs, etc

I checked the KIP and it didn't specify what the default should be, so I figured I could freely choose null (see [KAFKA-9559](https://issues.apache.org/jira/browse/KAFKA-9559))

Reply:
Woah. Good catch.